shared/actions/chat2/index.tsx

```diff
@@ -2534,6 +2534,27 @@ const joinConversation = async (_: TypedState, action: Chat2Gen.JoinConversation
   )
 }
 
+const fetchUserBio = async (state: TypedState, action: Chat2Gen.SelectConversationPayload) => {
+  const {conversationIDKey} = action.payload
+  const meta = Constants.getMeta(state, conversationIDKey)
+  const otherParticipants = Constants.getRowParticipants(meta, state.config.username || '').toArray()
+  if (otherParticipants.length === 1) {
+    // we're in a one-on-one convo
+    const username = otherParticipants[0]
+    if (state.users.infoMap.get(username, {bio: undefined}).bio) {
+      return // don't re-fetch bio if we already have one cached
+    }
+
+    const userCard = await RPCTypes.userUserCardRpcPromise({useSession: true, username})
```

Comment:
i think these actions actually belong in the user and not chat. this thing should just do
`UsersGen.createUpdateBio({username})`

Reply:
the users saga will listen for that action and make the call